### DISCUSSION TITLE
Unable to build a new mail server

### DISCUSSION THREAD

**Discussion by robustedge** [postalserver/postal#1996] [Help with using Postal]
Hi,

I did a fresh installation & after creating the organization unable to build a new mail server. After adding the name when I am clicking build server nothing is happending.
I rebooted the server & restarted the postal but no luck. Am I missing something?

Please help.

Thank you

**Comment by willpower232** [comment]
What is your `postal status`? Is everything up and running?

**Reply by robustedge** [reply]
Hi Will,
Thank you for your reply.
Here is the output of postal status:
root@ubuntu:~# postal status
      Name                     Command               State   Ports
------------------------------------------------------------------
postal_cron_1       /docker-entrypoint.sh post ...   Up
postal_requeuer_1   /docker-entrypoint.sh post ...   Up
postal_smtp_1       /docker-entrypoint.sh post ...   Up
postal_web_1        /docker-entrypoint.sh post ...   Up
postal_worker_1     /docker-entrypoint.sh post ...   Up

**Reply by willpower232** [reply]
Hopefully you will be able to see an error, either in `postal logs web` or by `postal bash web` and finding the log file in that container.

Off the top of my head, I'd imagine that you need to add the second database permission for access to databases starting with `postal-%`
